server/pkg/webserver/service/importer/importer.go

```diff
@@ -22,15 +28,48 @@ type ImportResult struct {
 }
 
 type ActionResult struct {
-	Results []Task `json:"results"`
-	Msg     string `json:"msg"`
+	Results []TaskInfo `json:"results"`
+	Msg     string     `json:"msg"`
+}
+
+func GetNewTaskDir() string {
+	taskID := GetTaskMgr().NewTaskID()
+	taskDir := filepath.Join(config.Cfg.Web.TasksDir, taskID)
+	return taskDir
+}
+
+func CreateConfigFile(dir string, config importconfig.YAMLConfig) error {
+	fileName := "config.yaml"
+	err := utils.CreateDir(dir)
+	if err != nil {
+		return err
+	}
+	path := filepath.Join(dir, fileName)
+	// erase user information
+	address := *config.NebulaClientSettings.Connection.Address
+	user := *config.NebulaClientSettings.Connection.User
+	password := *config.NebulaClientSettings.Connection.Password
+	*config.NebulaClientSettings.Connection.Address = ""
+	*config.NebulaClientSettings.Connection.User = ""
+	*config.NebulaClientSettings.Connection.Password = ""
+	outYaml, err := yaml.Marshal(config)
+	err = os.WriteFile(path, outYaml, 0644)
+	if err != nil {
+		zap.L().Warn("write"+path+"file error", zap.Error(err))
+		return err
+	}
+	defer func() {
+		*config.NebulaClientSettings.Connection.Address = address
+		*config.NebulaClientSettings.Connection.User = user
+		*config.NebulaClientSettings.Connection.Password = password
+	}()
```

Comment:
It's need `defer`? It's already at the end of func.

Reply:
If err happen，it also need to do